File: myapp/parent/Step50_LambdaLoop.cs
using sfn_ut.sfn.lambda;

namespace sfn_ut.myapp.parent;

public class Step50_LambdaLoop : SfnLambda<ParentPayload> {
  public override ParentPayload Execute(ParentPayload payload) {
    payload.AsyncOpStarted = true;
    return payload;
  }
}

File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/parent/Step50_LambdaLoop.java
package com.sfnut.myapp.parent;

import com.sfnut.sfn.lambda.SfnLambda;

public class Step50_LambdaLoop extends SfnLambda<ParentPayload> {
    @Override
    public ParentPayload execute(ParentPayload payload) {
        payload.setAsyncOpStarted(true);
        return payload;
    }
}
